GitSpace/Sources/ViewModels/TagViewModel.swift

```diff
@@ -22,11 +22,11 @@ final class TagViewModel: ObservableObject {
     @MainActor
     func requestTags() async -> Void {
         do {
+            self.tags.removeAll()
```

Comment:
```suggestion
            tags.removeAll()
```
[Airbnb Swift Style Guide](https://github.com/airbnb/swift#omit-self)를 참고하면, self 는 제거되어도 무방하다고 생각합니다!